crates/red_knot_python_semantic/src/types/call/bind.rs

```diff
@@ -333,27 +694,35 @@ impl<'db> CallableBinding<'db> {
 }
 
 /// Binding information for one of the overloads of a callable.
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug)]
 pub(crate) struct Binding<'db> {
     /// Return type of the call.
     return_ty: Type<'db>,
 
-    /// Bound types for parameters, in parameter source order.
-    parameter_tys: Box<[Type<'db>]>,
+    /// The formal parameter that each argument is matched with, in argument source order, or
+    /// `usize::MAX` if the argument was not matched to any parameter.
+    argument_parameters: Box<[Option<usize>]>,
```

Comment:
Does the field documentation need to be updated to reflect that change of using `None` instead of `usize::MAX`?

Reply:
It sure does! Thanks, good catch